src/screens/Platform/App.js

```diff
@@ -1,17 +1,21 @@
 // @flow
 
-import React from "react";
+import React, { useEffect } from "react";
 import { usePlatformApp } from "@ledgerhq/live-common/lib/platform/PlatformAppProvider";
 import type { StackScreenProps } from "@react-navigation/stack";
-import { useTheme } from "@react-navigation/native";
+import { useNavigation, useTheme } from "@react-navigation/native";
 import TrackScreen from "../../analytics/TrackScreen";
 import WebPlatformPlayer from "../../components/WebPlatformPlayer";
 
 const PlatformApp = ({ route }: StackScreenProps) => {
   const { dark } = useTheme();
   const { platform: appId, ...params } = route.params;
+  const { setParams } = useNavigation();
   const { manifests } = usePlatformApp();
   const manifest = manifests.get(appId);
+  useEffect(() => {
+    manifest.name && setParams({ name: manifest.name });
```

Comment:
When navigating to this screen, we don't necessarily have the manifest.name as we just need the `id` and not the manifest itself, so here we ensure that the title (`name`) displayed at the top of the screen is correct.